Sources/SyncDataProviders/Credentials/CredentialsProvider.swift

```diff
@@ -145,7 +168,6 @@ public final class CredentialsProvider: DataProvider {
                         }
                     default:
                         saveError = error
-                        break
```

Comment:
Interestingly, this one tripped SwiftLint's unnecessary break rule, but the other cases didn't. Is it because this is the `default` block?